src/.nuget/Microsoft.NETCore.Native/Microsoft.NETCore.Native.pkgproj

```diff
@@ -1,50 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
-  <PropertyGroup>
-    <SkipPackageFileCheck>true</SkipPackageFileCheck>
-    <SkipValidatePackage>true</SkipValidatePackage>
+
+  <Import Project="Microsoft.NETCore.Native.props" />
+
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), packaging.props))\packaging.props" />
+
+  <PropertyGroup Condition="'$(PackageTargetRuntime)' == ''">
     <PackagePlatforms>x64;x86;arm64;arm;</PackagePlatforms>
-    <OutputPath>$(PackagesOutputPath)</OutputPath>
-    <IncludeRuntimeJson>true</IncludeRuntimeJson>
+    <IsLineupPackage Condition="'$(IsLineupPackage)' == ''">true</IsLineupPackage>
   </PropertyGroup>
-  <ItemGroup>
-    <ProjectReference Include="linux\Microsoft.NETCore.Native.pkgproj">
-      <Platform>amd64</Platform>
-    </ProjectReference>
-    <ProjectReference Include="debian\Microsoft.NETCore.Native.pkgproj">
-      <Platform>amd64</Platform>
-    </ProjectReference>
-    <ProjectReference Include="fedora\23\Microsoft.NETCore.Native.pkgproj">
-      <Platform>amd64</Platform>
-    </ProjectReference>
-    <ProjectReference Include="fedora\24\Microsoft.NETCore.Native.pkgproj">
-      <Platform>amd64</Platform>
-    </ProjectReference>
-    <ProjectReference Include="opensuse\13.2\Microsoft.NETCore.Native.pkgproj">
-      <Platform>amd64</Platform>
-    </ProjectReference>
-    <ProjectReference Include="opensuse\42.1\Microsoft.NETCore.Native.pkgproj">
-      <Platform>amd64</Platform>
-    </ProjectReference>
-    <ProjectReference Include="rhel\Microsoft.NETCore.Native.pkgproj">
-      <Platform>amd64</Platform>
-    </ProjectReference>
-    <ProjectReference Include="ubuntu\14.04\Microsoft.NETCore.Native.pkgproj">
-      <Platform>amd64</Platform>
-    </ProjectReference>
-    <ProjectReference Include="ubuntu\16.04\Microsoft.NETCore.Native.pkgproj">
-      <Platform>amd64</Platform>
-    </ProjectReference>
-    <ProjectReference Include="ubuntu\16.10\Microsoft.NETCore.Native.pkgproj">
-      <Platform>amd64</Platform>
-    </ProjectReference>
-    <ProjectReference Include="alpine\3.4.3\Microsoft.NETCore.Native.pkgproj">
-      <Platform>amd64</Platform>
-    </ProjectReference>
-    <ProjectReference Include="osx\Microsoft.NETCore.Native.pkgproj">
-      <Platform>amd64</Platform>
-    </ProjectReference>
-  </ItemGroup>
+
+  <Import Condition="'$(_packageTargetOSGroup)' != ''" Project="$(MSBuildThisFileDirectory)runtime.$(_packageTargetOSGroup).$(MSBuildProjectName).props" />
```

Comment:
Not every project has a `runtime.$(_packageTargetOSGroup).$(MSBuildProjectName).props` file.  I could also condition on that existing, but then I was worried that we would hide a valid missing file if we're bringing up a new osgroup support or something.